### PR TITLE
Feature/vertical orientation prop #162

### PR DESCRIPTION
**Pull request by udaypydi**
- [ ] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)
- [ ] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`

### CODE REVIEW

**Comment by udaypydi** [comment]
@piotr-s-brainhub This is a PR for vertical orientation of the carousel. Please verify.

Thanks!

**Comment by piotr-s-brainhub** [comment]
Thanks @udaypydi for this PR.
Could you also add an example to README and docs/ demo?

**Comment by piotr-s-brainhub** [comment]
thanks @udaypydi for an example in README, now I'm waiting for an example in the demo/ docs (maybe you're already working on it)

**Comment by udaypydi** [comment]
@piotr-s-brainhub yeah i am working on it. Will push it in a while.

File: docs/examples/vertical.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Vertical Carousel
+You can pass a vertical prop to rotate the carousel in a vertical direction. Default value of vertical is false.
```
Comment (piotr-s-brainhub) [inline docs/examples/vertical.md]:
```suggestion
You can pass a `vertical` prop to rotate the carousel in a vertical direction. The default value of `vertical` is false.
```

**Comment by piotr-s-brainhub** [comment]
@udaypydi 

I opened https://github.com/brainhubeu/react-carousel/pull/252 instead because deployment to a testing environment works only from the `brainhubeu` branches

**Pull request merged**